exporter/datadogexporter/factory.go

```diff
@@ -283,11 +283,21 @@ func (f *factory) createMetricsExporter(
 		cancel()
 		return nil, fmt.Errorf("failed to start trace-agent: %w", err)
 	}
-	var statsOut chan []byte
+	acfg, err := newTraceAgentConfig(ctx, set, cfg, hostProvider)
+	if err != nil {
+		cancel()
+		return nil, err
+	}
+	statsToAgent := make(chan *pb.StatsPayload)
+	statsWriter := writer.NewStatsWriter(acfg, statsToAgent, telemetry.NewNoopCollector())
```

Comment:
Why a noop collector? Is it because statsd submission won't work as expected here?

Reply:
Yeah I believe that's why? Although I was just copying the original existing behavior which uses a noop collector in `newTraceAgent`